### PR TITLE
[docs] PR for Add documentation for cert-manager integration

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-5361

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@subramanian-neelakantan Could you, please, review this doc PR ?

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 1]
```diff
@@ -367,6 +367,50 @@ When you create a universe, you can enable TLS using certificates provided by Ha
 
 You can also edit TLS settings for an existing universe by navigating to **Universes**, opening a specific universe, clicking **Actions > Edit Security > Encryption in-Transit** to open the **TLS Configuration** dialog, and then modifying the required settings.
 
+## Kubernetes cert-manager
+
+For a universe created on Kubernetes, YugabyteDB Anywhere allows you to configure an existing running instance of the [cert-manager](https://cert-manager.io/) as a TLS certificate provider for a cluster, assuming that the following criteria are met:
+
+- The cert-manager is running in the Kubernetes cluster.
+- A root or intermediate CA (either self-signed or external) is already configured on the cert-manager. The same root certificate file must be prepared for upload to YugabyteDB Anywhere.
+- An Issuer or ClusterIssuer Kind of the cert-manager resource is prepared and is ready to issue certificates using the previously-mentioned root or intermediate certificate.
```
Comment (subramanian-neelakantan) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:376]: Should we reword this sentence a bit? Something like:
An Issuer or ClusterIssuer Kind of resource is configured on cert-manager and is ready to issue certificates ...

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 2]
```diff
@@ -367,6 +367,50 @@ When you create a universe, you can enable TLS using certificates provided by Ha
...
+
+During the universe creation, you can enable TLS certificates issued by the cert-manager, as follows:
+
+1. Upload the root certificate to YugabyteDB Anywhere:
+
+   - Prepare the root certificate in a file (for example, `root.crt`).
+   - Navigate to **Configs > Security > Encryption in Transit** and click **Add Certificate**.
+   - On the **Add Certificate** dialog shown in the following illustration, select **K8S cert-manager**:
+
+     ![img](/images/yp/security/kubernetes-cert-manager.png)
```
Comment (subramanian-neelakantan) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:386]: This image has a typo in the 'test-k8s-cret' :). Can I share another screen shot to upload here?

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md [hunk 3]
```diff
@@ -367,6 +367,50 @@ When you create a universe, you can enable TLS using certificates provided by Ha
...
+
+   - In the **Certificate Name** field, enter a meaningful name for your certificate configuration.
+   - Click **Upload Root Certificate** and select the root certificate file that you prepared.
+   - Click **Add** to make the certificate available.
+
+2. Configure the Kubernetes-based cloud provider by following instructions provided in [Configure region and zones](../../configure-yugabyte-platform/set-up-cloud-provider/kubernetes/#configure-region-and-zones). In the **Add new region** dialog, you would be able to specify the Issuer name or the ClusterIssuer name for each zone. Since an Issuer Kind is a Kubernetes namespace-scoped resource, the zone definition should also set the **Namespace** field value if an Issuer Kind is selected.
```
Comment (subramanian-neelakantan) [inline docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:392]: Adding an illustration here would be nice. Can I share a screen shot to upload here too?

**Review by subramanian-neelakantan — COMMENTED** [review]
Looks good overall to me. Just a few suggestions on the screen shots.

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:392]: @subramanian-neelakantan Yes, please, share the screenshot

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:376]: Changed

File: docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md:392]: Added screenshot

**Pull request merged**